modules/specialized-guides/pages/large-deployments/hub-reqs.adoc

```diff
@@ -4,22 +4,34 @@
 To set up a Hub installation, you require:
 
 * One central {productname} Server, which acts as the Hub Server.
-* One or more additional {productname} Servers, registered to the Hub as clients.
-    This document refers to these as peripheral servers.
+* One or more additional {productname} Servers, registered to the Hub.
+  This document refers to these additional servers as peripheral servers.
 * Any number of clients registered to the peripheral servers.
-* Ensure the Hub Server and all peripheral servers are running {productname}{nbsp}4.1 or higher.
+* Ensure the Hub Server and all peripheral servers are running {productname}{nbsp}5.0 or higher.
+
+
+
+== Hub Server
+
+Deploy the central {productname} Server, which acts as the Hub Server, with [command]``mgradm``.
+For more information about deploying with [command]``mgradm``, see xref:installation-and-upgrade:container-deployment/deployment-as-a-container.adoc[].
 
 [NOTE]
 ====
 The Hub Server must not have clients registered to it.
 Clients should only be registered to the peripheral servers.
 ====
 
+
+
 == Peripheral Servers
 
+Deploy the peripheral servers with [command]``mgradm``.
+For more information about deploying with [command]``mgradm``, see xref:installation-and-upgrade:container-deployment/deployment-as-a-container.adoc[].
+
 Peripheral servers must be registered to the Hub Server as clients.
-When you register the peripheral servers, assign them the appropriate ``SUSE Manager Server`` software channel as their base channel.
-Additionally, they must be registered to the Hub Server directly, do not use a proxy.
+When you register the peripheral servers, assign them the appropriate [guimenu]``{productname} Server`` software channel as their base channel.
+The peripheral servers must be registered to the Hub Server directly, do not use a proxy.
```

Comment:
Peripheral server must be registered as _foreign_ clients, that means the standard registration methods can't be used. Also, there are no channels involved.

To register the peripheral server, use mgradm command on the peripheral server host:
```
mgradm hub register --api-server <hub fqdn> --api-user <hub admin> --api-password <hub admin password>
```

Optionally, it is possible to register the podman container host as standard salt minion and manage it from SUMA.

Reply:
Question @nadvornik , if we onboard both the container and the host, it will trigger a merge of registered systems as it happens with proxy container and proxy host? If so, that would affect somehow to Hub functionality?

Reply:
There is no merging, they will stay separate.
Only the foreign client will have the "peripheral server" entitlement so the Hub API will use only this one.